packages/@webex/plugin-meetings/src/multistream/remoteMediaGroup.ts

```diff
@@ -66,6 +67,38 @@ export class RemoteMediaGroup {
     return [...this.unpinnedRemoteMedia, ...this.pinnedRemoteMedia];
   }
 
+  /**
+   * fixCsis
```

Comment:
Comments need to be a bit better as this is a public JS-SDK API that will be used potentially by other apps.
Also I'm not a big fan of the name "fixCsis" - the word "fix" can have multiple meanings (like fixing something that's broken). Maybe we can name it "setRemoteMediaCsis" or just "setCsis"

Reply:
actually we already have a `setRemoteVideoCsi()` API on RemoteMediaManager, but that one allows only calling it on "memberPanes" - so on the RemoteMedia that are by design "recever selected" on the current layout. Our new API is supposed to be called only on RemoteMedia objects that are part of an active speaker group, maybe this needs to be reflected in the API name somehow?